algosdk/v2client/algod.py

```diff
@@ -1,14 +1,37 @@
 import base64
 import json
+from typing import (
+    Any,
+    Dict,
+    Final,
+    Iterable,
+    List,
+    Mapping,
+    Optional,
+    Sequence,
+    Tuple,
+    Union,
+    cast,
+)
 import urllib.error
 from urllib import parse
 from urllib.request import Request, urlopen
 
 from algosdk import constants, encoding, error, transaction, util
 
+AlgodResponseType = Union[Dict[str, Any], bytes]
```

Comment:
`AlgodResponseType` is a result of the fact that the method `algod_request()` returns a dict when its `response_format == 'json'` (this is the default) and otherwise `bytes`